rambo/app.py

```diff
@@ -354,6 +354,23 @@ def up(ctx=None, provider=None, vagrant_cwd=None, vagrant_dotfile_path=None):
               'variable, and is not in the providers list. Did you '
               'have a typo?' % provider)
 
+    if guest_os:
+        print(guest_os)
+        set_env_var('guest_os', str(guest_os))
+        if guest_os not in GUEST_OSES:
+            msg = 'Guest OS "{}" is not in the guest OSes list. Did you have a typo? Here is as list of avalible guest OSes:\n'
+            msg = msg.format(guest_os)
+            for os in GUEST_OSES:
+                msg = msg + '{}\n'.format(os)
+            abort(msg)
+    elif get_env_var('GUEST_OS') and get_env_var('GUEST_OS') not in GUEST_OSES:
```

Comment:
This is specific to using env vars AND the API at the same time, I'm pretty sure. I'm not gong to test this right now. After a quick discussion with @smitty42, I see no reason to keep it. That can be in a follow up ticket though.